src/components/table/__spec__.js

```diff
@@ -928,6 +928,18 @@ describe('Table', () => {
         expect(instance.tableContent.get(0)).toEqual(instance.props.children.get(0));
         expect(instance.tableContent.get(1)).toEqual(instance.emptyRow);
       });
+
+      describe('when a customEmptyRow has been provided', () => {
+        test('the custom empty row is rendered instead of the default', () => {
+          let data = Immutable.fromJS([]),
```

Comment:
If a variable is not intended to be reassigned it should be declared with `const`